clippy_lints/src/matches.rs

```diff
@@ -664,6 +686,25 @@ fn check_match_as_ref(cx: &LateContext<'_, '_>, ex: &Expr, arms: &[Arm], expr: &
     }
 }
 
+fn check_pats_wild_match(cx: &LateContext<'_, '_>, _ex: &Expr, arms: &[Arm], _expr: &Expr) {
```

Comment:
Don't pass `ex` and `expr` into this function since it seems you aren't using them. The function header could just be `check_pats_wild_match(cx: &LateContext<'_, '_>, arms: &[Arm])`.